pytest_fixtures/core/sat_cap_factory.py

```diff
@@ -176,21 +194,23 @@ def large_capsule_configured(large_capsule_host, target_sat):
 
 
 @pytest.fixture(scope='module')
-def module_capsule_configured(module_capsule_host, module_target_sat):
+def module_capsule_configured(request, module_capsule_host, module_target_sat):
     """Configure the capsule instance with the satellite from settings.server.hostname"""
-    module_capsule_host.capsule_setup(sat_host=module_target_sat)
+    if not request.config.option.n_minus:
```

Comment:
Could you explain what this check is about?

Reply:
Here, If the `--n-minus` option is given to pytest command for collecting / executing all capsule tests, but since the capsule is already configured with satellite and passed in from capsule conf, we dont need to reconfigure the capsule. 

The condition here skips the capsule configuration for n-minus testing!